src/components/VideoQueueItem.vue

```diff
@@ -101,8 +97,8 @@ export default {
 		display: none;
 	}
 }
-.actions {
-	margin-left: 4px;
+.full-height {
+    height: 100%;
 }
```

Comment:
No need to make a new css class, you can use the `fill-height` property on vuetify components